Adding second footprint for 3V3 voltage regulator

jamesSaporito opened the pull request:
The voltage regulator that was originally used isn't available at the moment. To provide more options, a second footprint was added to the PCB for an alternative regulator that has a different pinout. Important to note that only one regulator should be used.

Also, adjusted .gitignore for Mac files and KiCad auto-generated backups.

jamesSaporito commented:
> Fairly neat addition, Only thing that jumps out to me is the 5v trace width on U9 isn't the same width as it is for the rest of the 5V rail. ![image](https://user-images.githubusercontent.com/1392296/194899995-d9d60a61-d29c-4ab2-8d55-9d0f43d5c93e.png)
> 
> And _MAYBE_ move the label for U9 to above the tab just so the labels visability is consistant with the rest of the labels. (So its not hidden if the part is soldered down) But thats just a visibity issue not really a game breaker.
> 
> No complaints on the schematic, was the same method I did last night in discord. I don't know what your personal reasoning for using a global label for GND in the power section but my reasoning when I did the same last night was because I feel it was more "readable" signalling ath either of the two regs could be used then simply tying the GND.
> 
> The .gitignore changes might be best in a sep pull request - don't see an issue adding ignoring .DS_Store but you are also adding an ignore for *backups

I can adjust the trace to be the same width as the others. If the U9 label is bothering you then I can adjust the location for that as well. Probably best to match all the other labels visibility. The addition of `*backups` was to get rid of all the auto-generated backup files/directories that KiCad creates. Perhaps it is too vague. I'll split out the .gitignore changes as well if you think it's best in a separate PR.

scottbez1 commented on electronics/view_base/view_base.kicad_pcb:
```diff
@@ -84,95 +84,95 @@
+  (net 82 "unconnected-(U3-Pad10)")
+  (net 83 "unconnected-(U3-Pad11)")
+  (net 84 "unconnected-(U3-Pad12)")
+  (net 85 "unconnected-(U3-Pad15)")
+  (net 86 "unconnected-(U4-Pad13)")
+  (net 87 "unconnected-(U5-Pad19)")
+  (net 88 "unconnected-(U7-Pad1)")
+  (net 89 "5V")
+  (net 90 "3V3")
```
Do you know why all the nets were re-numbered? I tried replicating a similar change (add a new regulator, wire everything up in the schematic) and I don't see net renumbering taking place. It's not a blocker, but it worries me a little bit about _why_ it happened, and makes the diff harder to review.

jamesSaporito replied:
That's a great question. I honestly don't know what happened there. If you tested it out and didn't experience the same, then it was most likely something I did by accident though. Will fix it.

jamesSaporito replied:
@scottbez1 I was able to reproduce this tonight. The problem was the global label `5V` that I added, which seems to "overwrite" the standard `+5V` symbol that I should have used instead. If you look at the diff you can see that `+5V` is gone and the `5V` has "replaced" it. So I guess it forces the nets to be re-numbered when doing that. When adding a random global label, it did not re-number all of the nets, just added to the list.

Would you like me to redo this without altering the original nets?

scottbez1 replied:
That's good to know - thanks for looking into it! No need to redo it, this is fine

scottbez1 reviewed (commented):
Cool, thanks for the PR! A couple thoughts:
- What's the reason for using global labels for power nets rather than the standard symbols (which are already global)?
- I'm wondering about the net renumbering (see inline comment) - maybe related to the added global labels?
- trace width - would be good to keep this consistent as mentioned

jamesSaporito commented:
> Cool, thanks for the PR! A couple thoughts:
> 
> * What's the reason for using global labels for power nets rather than the standard symbols (which are already global)?
> * I'm wondering about the net renumbering (see inline comment) - maybe related to the added global labels?
> * trace width - would be good to keep this consistent as mentioned

Thanks for reviewing!

No good reason for using global labels, I was attempting to make it easier to read in my mind, but I think it just made it messier. The global labels do seem to be a direct cause of the net re-numbering as well.

Went ahead and fixed the trace widths and adjusted the schematic to just use the standard power symbols instead. Let me know if you'd like me to redo this to preserve the original net numberings.

scottbez1 commented:
Thanks again for the PR! FYI I did a little cleanup in 04475a8810adecb0ec262d04037c58ef46f9b364 - I moved the LDOs so they mirror each other on the board, and updated the footprints to label the IN/GND/OUT pins on each to hopefully make the pinout distinction more obvious. Also added part numbers for the new LDO U9 and notes on both mentioning to only populate one or the other. Also a little bit of tidying the schematic.
![Screenshot from 2022-10-10 21-15-37](https://user-images.githubusercontent.com/414890/194995768-a138bce4-a454-41be-9341-1b05dd2efbec.png)

jamesSaporito commented:
@scottbez1 Thanks for approving! Love the cleanup and clarifications you made.
